frontend/src/lib/components/TaxonomicFilter/InfiniteSelectResults.tsx

```diff
@@ -38,8 +38,8 @@ function CategoryPill({
             type={isActive ? 'primary' : canInteract ? 'option' : 'muted'}
             data-attr={`taxonomic-tab-${groupType}`}
             onClick={canInteract ? onClick : undefined}
-            weight="normal"
-            aria-disabled
```

Comment:
@EDsCODE Looks like this was added in https://github.com/PostHog/posthog/pull/22385. What was the reason?